mod_wsbridge/mod_wsbridge.c

```diff
@@ -384,6 +417,67 @@ websocket_write_back(struct lws *wsi_in, enum lws_write_protocol type, char *str
 	return n;
 }
 
+void add_content_type(private_t *tech_pvt, cJSON* json) {
+	if (cJSON_GetObjectItem(json, "content-type")) {
+		cJSON_DeleteItemFromObject(json, "content-type");
+	}
+	cJSON_AddItemToObject(json, "content-type", cJSON_CreateString(tech_pvt->content_type));
+}
+
+int is_mute_event(char* event, char* method) {
+	return !strcmp(event, "websocket:media:update") && !strcmp(method, "update");
+}
+
+void reset_ws_write_indexes(private_t *tech_pvt) {
+	switch_mutex_lock(tech_pvt->write_mutex);
+	tech_pvt->write_count = 0;
+	tech_pvt->write_start = 0;
+	tech_pvt->write_end = 0;
+	switch_mutex_unlock(tech_pvt->write_mutex);
+}
+
+void on_event(private_t *tech_pvt, cJSON* json) {
+	char* event = NULL;
+	char* method = NULL;
+	int active = 0;
+	event = cJSON_GetObjectItem(json, "event")->valuestring;
+	method = cJSON_GetObjectItem(json, "method")->valuestring;
+	if (is_mute_event(event, method)) {
+		active = cJSON_GetObjectItem(json, "active")->valueint;
```

Comment:
This can crash if the the value is not an int.  For example, if the request has "active":xxx it will crash. We should verify that "active" contains a int value before interpreting it as an int.